draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
OK, got it. In this case I think the text that says "a=msid must stay the same" needs to change:

      For RtpTransceivers that are not stopped, the "a=msid" line MUST
      stay the same if they are present in the current description.